_opensearch/rest-api/explain.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+title: Explain
+parent: REST API reference
+nav_order: 140
+---
+
+# Explain
+Introduced 1.0
+{: .label .label-purple }
+
+Wondering why a specific document ranks higher (or lower) for a query? You can use the explain API for an explanation of how the relevance score (`_score`) is calculated for every result.
+
+OpenSearch uses a probabilistic ranking framework called [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25) to calculate relevance scores. Okapi BM25 is based on the original [TF/IDF](http://lucene.apache.org/core/4_0_0/core/org/apache/lucene/search/package-summary.html#scoring) framework used by Apache Lucene.
+
+The explain API is an expensive operation in terms of both resources and time. We recommend you don't use it for production workloads.
```

Comment:
I think we can reword this second sentence a bit -- surely it's appropriate to use it for one-off requests. Maybe something like "On production clusters, we recommend using it sparingly for the purpose of troubleshooting."?